src/App.js

```diff
@@ -11,12 +11,20 @@ import InfoButton from './components/Buttons/InfoButton'
 import PadSection from './components/Pad/PadSection'
 
 const samplePlayer = new SamplePlayer(padsData)
+const focusElementAfterLoad = element => {
```

Comment:
Maybe move this to the bottom of the file as a function declaration? Seems generic enough to justify putting it in `util.js` as well.